Actually support eabi arm in the bindings generator.
Patch by Arno Rehn. Stolen from upstream.
svn diff -c 1124282 svn://anonsvn.kde.org/home/kde/

Index: branches/KDE/4.4/kdebindings/generator/generatorpreprocessor.cpp
===================================================================
--- a/generator/generatorpreprocessor.cpp	(revision 1124281)
+++ b/generator/generatorpreprocessor.cpp	(revision 1124282)
@@ -77,6 +77,14 @@
     exportMacro->variadics = false;
     m_topBlock->setMacro(exportMacro);
 
+#if (defined(QT_ARCH_ARM) || defined (QT_ARCH_ARMV6)) && !defined(QT_NO_ARM_EABI)
+    exportMacro = new rpp::pp_macro;
+    exportMacro->name = IndexedString("__ARM_EABI__");
+    exportMacro->function_like = false;
+    exportMacro->variadics = false;
+    m_topBlock->setMacro(exportMacro);
+#endif
+
     // ansidecl.h will define macros for keywords if we don't define __STDC__
     exportMacro = new rpp::pp_macro;
     exportMacro->name = IndexedString("__STDC__");
